ndcube/ndcube.py

```diff
@@ -255,20 +259,33 @@ def dimensions(self):
         Returns a named tuple with two attributes: 'shape' gives the shape
         of the data dimensions; 'axis_types' gives the WCS axis type of each dimension,
         e.g. WAVE or HPLT-TAN for wavelength of helioprojected latitude.
+
+        """
+        return u.Quantity(self.data.shape, unit=u.pix)
+
+    @property
+    def world_axis_physical_types(self):
         """
+        Returns an iterable of strings describing the physical type for each world axis.
+
+        They should be names from the VO UCD1+ controlled
```

Comment:
I would re-write this section of the docstring to describe the output to a user rather than the description to developers from the APE.

Reply:
Good spot!  Hadn't looked at that since I pasted it into the file!